_lessons/python-scatter/05-mpi.md

```diff
@@ -148,7 +148,10 @@ mpiexec -n 8 python scatter.py
 
 ## How to use MPI to accelerate `scatter.py`
 
- * At the top: `from mpi4py import MPI`. This will initialise MPI. The number of processes is `nprocs`.
+ * `from mpi4py import MPI` at the top. This will initialise MPI.
+ * `comm = MPI_COMM_WORLD` sets the communicator.
+ * `nprocs = comm.Get_size()` stores the total number of processes.
+ * `pe = comm.Get_rank()` stores the actual rank of an process.
```

Comment:
```suggestion
 * `pe = comm.Get_rank()` gets the actual rank of a process.
```

Reply:
these lines get the comm/rank/... and store it an variable. So one could argue, but I am fine with the change.